ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -248,3 +258,15 @@ if ! wait "$!"; then
   # TODO: gather logs??
   exit 1
 fi
+
+# Exit normally if there is no day2 jobs
+if [ "${ADDITIONAL_WORKERS}" == "0" ]; then
+    echo "No additional workers requested"
+    exit 0
+fi
+# Extract the ignition file for additional workers if additional workers count > 0
+oc extract -n openshift-machine-api secret/worker-user-data-managed --keys=userData --to=- > "${SHARED_DIR}"/worker.ign
+
+echo -e "\nCopying ignition files into bastion host..."
+chmod 644 "${SHARED_DIR}"/*.ign
+scp "${SSHOPTS[@]}" "${SHARED_DIR}"/*.ign "root@${AUX_HOST}:/opt/html/${CLUSTER_NAME}/"
```

Comment:
Just a final thing @zniu1011 , can you put this snippet in https://github.com/openshift/release/blob/master/ci-operator/step-registry/upi/install/heterogeneous/upi-install-heterogeneous-commands.sh#L141 ?

I don't like this logic to stay here and since it applies well for upi on bm too, we can put it there for all (otherwise, some other agent workflows might need this change too)

Reply:
Hi @aleskandro, if it is ok that copy and overwrite "*.ign" files the second time in UPI day2 install for you, I will update it.

Reply:
yes, it's harmless

Reply:
@aleskandro, the update is done, the job is running fine after the updates too.